### DISCUSSION TITLE
is there any major benefit of running virtual machine in containerd instead of kvm+qemu?

### DISCUSSION THREAD

**Discussion by Greatz08** [lima-vm/lima#2393] [Q&A]
it was written that "The original goal of Lima was to promote [containerd]" so my question is what special benefit it gives which qemu doesnt and does it support kvm like qemu . Performance+features wise how much does it differ from qemu with kvm?

**Comment by afbjorklund** [comment]
I think you misunderstood the goal, Lima will run containerd in a VM* (using qemu with kvm, where available). Just that you don't _have_ to use containerd, you can also use other container runtimes or just use the virtual machine for other purposes.

\* except for with the WSL driver
